<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiblePassageDialog</class>
 <widget class="QDialog" name="BiblePassageDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bible Passage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox_bibles"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_books"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_chapter"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_verse"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
